force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,41 @@
+trigger AccountTrigger on Account (before insert, after insert) {
+    
+    if (Trigger.isBefore && Trigger.isInsert) {
+        for (Account acc : Trigger.new) {
+            if (acc.Type == null) {
+                acc.Type = 'Prospect';
+            }
+        }
+    }
+    
+    if (Trigger.isBefore && Trigger.isInsert) {
+        for (Account acc : Trigger.new) {
+            if (acc.ShippingStreet == null || acc.ShippingCity == null || acc.ShippingState == null || acc.ShippingPostalCode == null || acc.ShippingCountry == null) {
```

Comment:
Try to separate these fields into different if statements. The logic that you have there, if any field is null the record will not be updated.